releasenotes/notes/profiling-add-lock-with-f75908e35a70ab71.yaml

```
@@ -0,0 +1,3 @@
+features:
+  - |
+    profiling: captures lock usages with `with` context managers, e.g. `with lock:`
```

```suggestion
    profiling: captures lock usages with ``with`` context managers, e.g. ``with lock:``
```

nit, this is rST and not Markdown